spec/spec_helper.rb

```diff
@@ -17,6 +27,10 @@ module VERSION
   end
 end
 
+require 'rubocop/rake_task'
+RuboCop::RakeTask.new
+Rake::Task['rubocop'].invoke
```

Comment:
Not sure if that is a good idea. If we want to run rubocop, it should be just another line in travis.yml. It should not be executed when running specs.